input/size.go

```diff
@@ -533,6 +533,14 @@ func (twe *TxWeightEstimator) AddNestedP2WSHInput(witnessSize int) *TxWeightEsti
 	return twe
 }
 
+// AddTxOutput adds a known TxOut to the weight estimator.
+func (twe *TxWeightEstimator) AddTxOutput(txOut *wire.TxOut) *TxWeightEstimator {
+	twe.outputSize += txOut.SerializeSize()
```

Comment:
Why's this needed given the other helper functions for the standard output sizes?

Reply:
I guess this is also a move towards using the _empirical_ weight of transaction datum instead of the estimates which we've had issues with in the past.

Reply:
Just that we have the actual `TxOut` at this point. So felt more correct to use its exact size than threading through a type such that we could make an estimate.